beetmoverscript/utils.py

```diff
@@ -51,6 +51,16 @@ def write_file(path, contents):
         fh.write(contents)
 
 
+def shipping_a_release(action):
```

Comment:
I feel like this name doesn't return a Boolean. How about something like: `is_action_a_release_shipping`?